autokeras/hypermodels/heads.py

```diff
@@ -169,3 +170,102 @@ def build(self, hp, inputs=None):
 
     def get_adapter(self):
         return adapters.RegressionHeadAdapter(name=self.name)
+
+
+class SegmenterHead(head_module.Head):
```

Comment:
I think we should extend the ClassificationHead class. They share a lot in common.